### PR TITLE
Fixing404 wording

### PR DESCRIPTION
**Pull request by arivepr**
This card is to close issue RHCLOUD-14098

### CODE REVIEW

**Comment by rvsia** [comment]
@arivepr you have included `.DS_STORE` file. Please, remove it. You can also set up [global gitignore](https://docs.github.com/en/github/getting-started-with-github/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) to ignore it.

File: packages/components/src/Components/Dark/Dark.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
```
Comment (rvsia) [inline packages/components/src/Components/Dark/Dark.d.ts:1]: I think all these declaration type files are here by mistake, aren’t they?

Comment (arivepr) [reply packages/components/src/Components/Dark/Dark.d.ts:1]: I'm not entirely sure tbh, haven't touched these files myself.

**Comment by arivepr** [comment]
@rvsia tookout the DS_store file. It shows up on the local gitIgnore, don't know how it nudged itself in.

File: packages/components/src/Components/Dark/Dark.d.ts (hunk 1 above)
Comment (rvsia) [reply packages/components/src/Components/Dark/Dark.d.ts:1]: Yep, then it should not be here too.

**Review by Hyperkid123 — COMMENTED** [review]
@arivepr I think you had a very very old version of the repository locally and you did not clean it up after the update. None of the `*.d.ts` should be there. In fact the folder `/src/Components/*` no longer exists.

**Pull request closed without merging**